### PR TITLE
core: random: Factored out v_noise

### PR DESCRIPTION
**Pull request by fweik**
Description of changes:
 - Factored out v_noise function
 - Removed unused code

### CODE REVIEW

File: testsuite/python/dpd.py [hunk 1]
```diff
@@ -462,7 +462,7 @@ def test_momentum_conservation(self):
         momentum = np.matmul(s.part[:].v.T, s.part[:].mass)
         for _ in range(10):
             s.integrator.run(25)
-            np.testing.assert_array_less(np.zeros((3, 3)), np.abs(s.part[:].f))
+            np.testing.assert_almost_equal(np.zeros((3,)), np.sum(s.part[:].f))
```
Comment (jngrad) [inline testsuite/python/dpd.py:465]: shouldn't it be `np.sum(np.abs(s.part[:].f), axis=1)`?

Comment (fweik) [reply testsuite/python/dpd.py:465]: Why `abs`? I want to check that the total force is zero.

Comment (jngrad) [reply testsuite/python/dpd.py:465]: I misunderstood the test logic.

**Review by jngrad — APPROVED** [review]
LGTM

**Pull request merged**